packages/syntax-core/src/Checkbox/Checkbox.tsx

```diff
@@ -68,6 +70,12 @@ const Checkbox = ({
    * @defaultValue false
    */
   error?: boolean;
+  /**
+   * Whether or not there is an error with the input
+   *
+   * @defaultValue false
+   */
+  ariaValues?: ariaProps;
```

Comment:
@somethiiing I would avoid passing in an object of props when possible. Just use `aria-describedby` instead.

Reply:
Example as to why we want to avoid it:
![image](https://github.com/Cambly/syntax/assets/127199/2ab40ed0-4538-4169-8637-5b52c894581b)